src/com/goide/inspections/GoDuplicateReturnArgumentInspection.java

```diff
@@ -29,6 +30,7 @@
 public class GoDuplicateReturnArgumentInspection extends GoDuplicateArgumentInspection {
   @Override
   public void check(@Nullable GoSignature o, @NotNull ProblemsHolder holder) {
+    super.check(o, holder);
```

Comment:
are you sure? looks like o.getParamters will be checked twice and also they will be check even if `GoDuplicateArgumentInspection` is disabled.